examples/framework/preferences/shared/html/slider-template.html

```diff
@@ -0,0 +1,7 @@
+<div>
+    <label class="mpe-slider-label"></label>
+    <div class="mpe-slider">
+        <input aria-labelledby="text-size-label" class="flc-textfieldSlider-slider fl-slider">
```

Comment:
Should the value of `aria-labelledby` at line 4, 5 be `mpe-slider-label` rather than `text-size-label`?